src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -2513,6 +2514,20 @@ else if (elem.equals("false")) {
         tagProcessor.registerStaticTag(TimeTag.class, "millis_to_time", (attribute, object) -> {
             return new TimeTag(object.asLong());
         });
+
+        // <--[tag]
+        // @attribute <ElementTag.unaccented>
+        // @returns ElementTag
+        // @description
+        // Returns a modified version of the input text but with all the accents removed and replaced with their normalized counterparts.
+        // Note that not all characters have a normalized form, such as "æ", and will be unchanged.
+        // @example
+        // # Narrates "TqaaCwIMæ"
+        // - narrate <element[ⓉⓠäắÇẘℑℳæ].unaccented>
+        // -->
+        tagProcessor.registerStaticTag(ElementTag.class, "unaccented", (attribute, object) -> {
+            return new ElementTag(Normalizer.normalize(object.asString(), Normalizer.Form.NFKD).replaceAll("[\\u0300-\\u036f]", ""));

```

Comment:
Probably better to create and cache a `Pattern` - not as much of a difference since it's a static tag, but it'd still need to compile the regex on every first parse when done this way